src/GNCTestServer.jl

```diff
@@ -339,11 +343,8 @@ Arguments:
 Returns:
 - hist:        Initialized log of the simulation                         |  Matrix
 """
-function default_log_init(state, iterations)
-    hist = zeros(iterations, 4)
-    hist[1, 1:3] .= state.ω
-    hist[1, 4] = norm(state.ω)
-    return hist
+function default_log_init(state)
```

Comment:
When testing I get an error:
"no method matching default_log_init(::NamedTuple{(:r, :v, :q, :ω), NTuple{4, Vector{Float64}}}, ::Int64)"
on line 14 of the test file. There must be some function call that is not using these updated arguments although I can't seem to find where.

Reply:
After looking a little more, I think this comes from line 50 in the test.jl file where we call Simulator.socket_simulator(), but I could be wrong about that.